exercises/tournament/src/test/java/TournamentTest.java

```diff
@@ -0,0 +1,158 @@
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class TournamentTest {
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @Test
+    public void justTheHeaderIfNoInput() {
+        Tournament tournament = new Tournament();
+        assertEquals("Team                           | MP |  W |  D |  L |  P", tournament.printTable());
+    }
+
+    @Test
```

Comment:
This (and all the following tests) need to be igored. See our [POLICIES doc](https://github.com/exercism/java/blob/master/POLICIES.md#ignore-noninitial-tests) for a description of how :)